### PR TITLE
Removes dead code from Lens

### PR DESCRIPTION
**Pull request by codefromthecrypt**
Found some of this with IntelliJ

### CODE REVIEW

File: zipkin-lens/src/components/DependenciesPage/DependenciesGraph.tsx [hunk 1]
```diff
@@ -94,10 +94,6 @@ const reactSelectStyles = {
     ...base,
     cursor: 'pointer',
   }),
-  clearIndicator: (base: CSSProperties) => ({
-    ...base,
-    cursor: 'pointer',
-  }),
```
Comment (tacigar) [inline zipkin-lens/src/components/DependenciesPage/DependenciesGraph.tsx]: I think this code is alive.

Without this code the cursor does not change when hovering over the clear button.

without

With this code: 

with

File: zipkin-lens/src/slices/remoteServicesSlice.ts [hunk 2]
```diff
@@ -77,6 +77,4 @@ const remoteServicesSlice = createSlice({
   },
 });
 
-export const { clearRemoteServices } = remoteServicesSlice.actions;
```
Comment (tacigar) [inline zipkin-lens/src/slices/remoteServicesSlice.ts:80]: Thanks!
It looks like these codes can be removed as well :)

https://github.com/openzipkin/zipkin/blob/5562dd5b3677944027450f546fe4b9cc1b2ca5bd/zipkin-lens/src/slices/remoteServicesSlice.ts#L55-L59

https://github.com/openzipkin/zipkin/blob/5562dd5b3677944027450f546fe4b9cc1b2ca5bd/zipkin-lens/src/slices/spansSlice.ts#L55-L59

https://github.com/openzipkin/zipkin/blob/5562dd5b3677944027450f546fe4b9cc1b2ca5bd/zipkin-lens/src/slices/spansSlice.ts#L80

File: zipkin-lens/src/components/DependenciesPage/DependenciesGraph.tsx (hunk 1 above)
Comment (codefromthecrypt) [reply zipkin-lens/src/components/DependenciesPage/DependenciesGraph.tsx]: intellij analysis is imperfect :) I will put a comment that this is "not quite dead"

File: zipkin-lens/src/slices/remoteServicesSlice.ts (hunk 2 above)
Comment (codefromthecrypt) [reply zipkin-lens/src/slices/remoteServicesSlice.ts:80]: great

Comment (codefromthecrypt) [reply zipkin-lens/src/slices/remoteServicesSlice.ts:80]: I am curious on these.. why can I remove them? Maybe you can tell me for each below:

servicesSlice, autocompleteKeys - only loaded ever once, so no need to clear
autocompleteValues - currently no reducer; data is partitioned by 'autocompleteKey'
remoteServices, spans - reducer to remove; data partitioned by 'service'
dependencies - contains a clear reducer, the data is not really cache-able except what's defined in HTTP

Later I would like to refactor based on this conversation. I would like to delete code, just I don't understand when we need a reducer to clear and not. cc @anuraaga

Comment (tacigar) [reply zipkin-lens/src/slices/remoteServicesSlice.ts:80]: https://github.com/openzipkin/zipkin/blob/5562dd5b3677944027450f546fe4b9cc1b2ca5bd/zipkin-lens/src/components/DiscoverPage/SearchBar/SearchBar.tsx#L138-L149

As soon as service criterion is changed, the spans and remoteServices fetching will begin, and the previous spans and remoteServices will be cleared.
So I think there is no need to explicitly clear them.

Comment (codefromthecrypt) [reply zipkin-lens/src/slices/remoteServicesSlice.ts:80]: thanks!

**Comment by codefromthecrypt** [comment]
ok let me know anything else on this round

**Review by tacigar — APPROVED** [review]
Thank you so much! 😃

**Review by anuraaga — APPROVED** [review]

**Pull request merged**